wp-content/plugins/AB-Content-Alterer/ab-content-alterer.php

```diff
@@ -2,10 +2,142 @@
 /*
 Plugin Name:  AB Content Alterer
 Description:  WordPress Content Alterer: Change Unbounce and Leadpages in content to <strong>Instapage</strong>
-Version:      0.1.0
+Version:      0.1.1
 Author:       andrzej.bialowas@instapage.com
 */
 defined('ABSPATH') or die('No script kiddies please!');
+
+class abContentAlererSettingsPage {
+    /**
+     * Holds the values to be used in the fields callbacks
+     */
+  private $options;
+
+  /**
+   * Start up
+   */
+  public function __construct() {
+    add_action('admin_menu', array( $this, 'add_plugin_page'));

```

Comment:
Follow PHP coding style for Website, use [] instead of array(). 
Don't use spaces after opening '[' and closing ']'